payload.adoc

```diff
@@ -206,6 +206,8 @@ is 0 is that it may be at either the address of the next instruction or
 current instruction depending on the exception cause, which can be
 inferred by the decoder without adding complexity to the encoder.)
 
+Where an interrupt or exception causes a privilege change, this changes comes into force from the start of the trap handler.  As such, the privilege value reported when *thaddr* is 0 will be the privilege level prior to any change.
```

Comment:
a tiny hiccup `this changes` -> `this change`

Reply:
Fixed, thanks